tools/ci/pr_preview.py

```diff
@@ -256,6 +267,32 @@ def delete_ref(self, refspec):
             full_ref
         ])
 
+    def get_pr_open_numbers(self):
+        """Returns a list of pull requests matching refs/prs-open/{pr}"""
+
+        refspec = 'refs/prs-open/*'
+
+        logger.info('Fetching all ref names matching "%s"', refspec)
+
+        output = subprocess.check_output([
+            'git',
+            '-c',
+            'credential.username={}'.format(self._token),
+            '-c',
+            'core.askPass=true',
```

Comment:
We now have at least three of these. Maybe create a helper method?

Reply:
Moved the joint `git -c ... -c ...` part into a helper.